starter-code/css/main.css

```diff
@@ -4,7 +4,165 @@ dark blue: #06365f
 light blue: #1c5380
 */
 
+/* --------- reset ---------- */
+
+
+
+html {
+
+  box-sizing: border-box;
+
+}
+
+*, *:before, *:after {
+
+  box-sizing: inherit;
+
+}
+
+
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
-  margin: 0;
+  margin: 0 auto;
+  background: url("/home/david/Documents/Ironhack/Week 1/Day 2/lab-css-instagram-clone/starter-code/images/bgtile.png");
+  background-repeat: repeat-x;
+  max-width: 900px;
+
+
+}
+
+/* --------- typography ---------- */
+
+.paragraph-style {
+  float: right;
+  text-align: left;
+  margin-top: -225px;
+  margin-right: 150px;
+ font-size: smaller;
+}
+
+h1{
+  float: left;
```

Comment:
Hmm don't make titles float hrom h1 tags. Better create a class and make it float from the class properties